src/tracer.js

```diff
@@ -80,6 +80,9 @@ export default class Tracer {
             } else {
                 fields.operationName = nameOrFields;
             }
+            if (fields.parent) {
+                fields.parent = fields.parent._imp;
```

Comment:
@yurishkuro Maybe a real implementation can explain my idea. Take a look at this https://github.com/CatTail/basictracer-javascript/blob/fa6e42d4228fa922ddc9e0c96ff204b1c17153cc/src/__tests__/index-test.js#L48